libs/detectors/x86/detector.py

```diff
@@ -1,3 +1,37 @@
+import json
+import os
+from pathlib import Path
+
+
+def get_or_create_model_json_file(config, device, camera_id):
+    """
+    If no model directory was created for a given device, the default values will be taken from the configuration file
+    and a model_<device>.json file will be created.
+    """
+    base_path = os.path.join(config.get_section_dict("App")["EntityConfigDirectory"], "sources", camera_id)
+    models_directory_path = os.path.join(base_path, "ml_models")
+    json_file_path = os.path.join(models_directory_path, f"model_{device}.json")
+
+    if not os.path.exists(json_file_path):
+        # Hypothesis: source config directory (base_path) should always exists.
+        if not os.path.exists(models_directory_path):
+            Path(models_directory_path).mkdir(parents=True, exist_ok=True)
```

Comment:
I think we can run this line without the ifs (as it's okay if it already exists and creates the parent's folders)

Reply:
You would still need the first if so I think it is okay this way. It would work without ifs but this is clearer I think

Reply:
Personally, I think this will save a lot of effort if a problem ever arises.